examples/GainPlugin/Source/jsui/src/Meter.js

```diff
@@ -37,10 +37,10 @@ class Meter extends Component {
     });
   }
 
-  _onMeasure(width, height) {
+  _onMeasure(e) {
```

Comment:
Nice. One thing that would be nice here is some doc or TS definition for a MeasureEvent or anything non-standard. As I understand it, for the other event types Typescript provides typings (provided Blueprint continues to confirm to the W3C) from the dom lib which users could rely on. 

https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.mouseevent.html

Reply:
Glad I held off with the components lib! This adds some significant but nice changes!

Reply:
Yea, we should definitely enumerate what these event objects carry. The goal is for the `SyntheticEvent` to roughly match the `UIEvent` spec, and the `SyntheticMouseEvent` to match `MouseEvent`, etc. So ideally we could just use the typescript defs there, but we're a long way away from parity, so I wonder if typescript would yell at us the whole time

Reply:
Yeah. I think it would be fine depending on what we use. For example I've actually used to dom CanvasRenderingContext2D typing in my code and everything works fine. In this case though maybe adding definitions for `SyntheticMouseEvent` and friends makes the most sense so that we write to those interfaces. Not sure how you feel about this for the core components. I'll hold off till this is merged and then drop you a PR ASAP.